docs/_future/getting_started_future.md

```diff
@@ -88,21 +89,32 @@ The application will take a few moments to set up and clone the repository. Once
+
+The starter template your app is based off of includes a "Hello World" example that demonstrates how to use the three main building blocks of next-generation Slack apps: [built-in](/bolt-js/future/built-in-functions) and [custom](/bolt-js/future/custom-functions) functions, [triggers](/bolt-js/future/triggers), and [workflows](/bolt-js/future/workflows).
+
+**At a quick glance:**
+1. **Functions** are what define the actions of your app
+2. **Workflows** are made up of **functions**
+3. **Triggers** are how **workflows** are executed
+
+If you'd like to test the "Hello World" example included in your template and see how functions, workflows, and triggers all tie together, read on! 📖
+
+> 💡 Now that it's been set up, this application can be used as starting point for you to build your own next-generation features. You can explore on your own and dive into the code yourself [here](https://github.com/slack-samples/bolt-js-starter-template/tree/future), or continue your journey by diving into [app manifests](/bolt-js/future/app-manifests) or learning how to add more [functions](/bolt-js/future/built-in-functions), [workflows](/bolt-js/future/workflows), and [triggers]() to your app!
 
 ---
 ### Set up your trigger {#setup-trigger}
 
-As mentioned, this app comes with preexisting functionality - it uses Functions, Workflows and a Trigger set up to kick things off. Let's run a command to initialize that Trigger via the CLI.
+Your app comes with a pre-built [workflow](https://github.com/slack-samples/bolt-js-starter-template/blob/future/manifest/workflows/sample-workflow.js) that uses a pre-written [function](https://github.com/slack-samples/bolt-js-starter-template/blob/future/manifest/functions/sample-function.js). This workflow is executed by a trigger defined in [this trigger configuration file](https://github.com/slack-samples/bolt-js-starter-template/blob/future/triggers/hello-world-trigger.json). We can run a command via the CLI to initialize this trigger so that we can run the workflow.
 
 First, make sure you're in the project directory in your command line: `cd my-app`
 
-Then, run the following command to create a Trigger:
+Then, run the following command to create a trigger:
 ```
-slack triggers create --trigger-def "triggers/hello-world-trigger.json"      
+slack trigger create --trigger-def "triggers/hello-world-trigger.json"      
 ```
 
-The above command will create a trigger for the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
+The above command will create a link trigger, which is run by clicking on a given URL, for the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
```

Comment:
```suggestion
The above command will create a link trigger belonging to the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
```

I think we can get rid of the "which is run by clicking". Let's save that for the section right after.